app/src/common/shared/com/igalia/wolvic/telemetry/TelemetryService.java

```diff
@@ -249,7 +250,7 @@ public static void receivedTab(@NonNull mozilla.components.concept.sync.DeviceTy
         @NonNull
         @Override
         public String getEntryName() {
-            return "";
+            return APP_NAME;
```

Comment:
I'll just use "android-components" which is what firefox for android uses as it looks more future proof

Reply:
If I'm not mistaken, it seems that Firefox for Android uses different values depending on the entry point for the authentication, so it seems fine to use a different one:

https://github.com/mozilla-mobile/firefox-android/blob/main/fenix/app/src/main/java/org/mozilla/fenix/components/accounts/FenixFxAEntryPoint.kt#L12

I have changed the name to `"wolvic"` (lowercase) to follow the same pattern.